test/functional/wallet_resendwallettransactions.py

```diff
@@ -68,6 +73,53 @@ def run_test(self):
         node.setmocktime(now + 36 * 60 * 60 + 600)
         peer_second.wait_for_broadcast([txid])
 
+        self.log.info("Chain of unconfirmed not-in-mempool txs are rebroadcast")
+        # This tests that the node broadcasts the parent transaction before the child transaction.
+        # The previous behavior is that the rebroadcasting would simply iterate mapWallet. As
```

Comment:
nit: I think `the previous behavior` stuff belongs to the commit message, not to the file content.

Reply:
Done